mix.exs

```diff
@@ -18,6 +18,7 @@ defmodule GTFSRealtimeViz.Mixfile do
 
   def application do
     [
+      apps: [:timex],
```

Comment:
I don't think this is necessary. My understanding is the new version of Elixir automatically includes the deps, and if you *don't* want one included, you put `runtime: false` in the deps list. Or, if you need an app that isn't a dependency (e.g. Logger), that goes in `extra_applications`.